dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/BasicDeploymentForm.scss

```diff
@@ -53,6 +53,7 @@
   }
   .slider-content {
     width: 30vw;
+    margin-left: 10px;
```

Comment:
Are we using `px`  in other margins? I'd say we use `em`, but I don't really know atm.

Reply:
we are, there is no standard for this. In this case I am using pixels because it's related to the size of the slidder circle (which is not relative to the size of the rest of the page)